packages/airnode-validator/src/config/config.ts

```diff
@@ -133,7 +135,7 @@ export const gatewaySchema = z.discriminatedUnion('enabled', [enabledGatewaySche
 export const enabledHeartbeatSchema = z
   .object({
     enabled: z.literal(true),
-    apiKey: z.string(),
+    apiKey: apiKeySchema,
```

Comment:
Heartbeat doesn't technically have such a restriction but I guess it's fine. Make sure to check with the ChainAPI guys. I presume they're using UUIDs for API keys but just to make sure.

Reply:
@andreogle does this work for you? I am enforcing our API keys (including heartbeat) to have length 30-120 characters.

Reply:
I'm fine with that. ChainAPI does use UUIDs for API keys